### PR TITLE
feat: add front-end dependency upgrades release notes

### PR DESCRIPTION
**Pull request by mritense**
Includes migration notes.

### CODE REVIEW

File: release-notes/major10/10.0.0/migration.md [hunk 1]
```diff
@@ -24,4 +24,99 @@ This page describes how to update Valtimo from the previous version to the curre
...
 
-       
+* **Dependency upgrades**
+
+  Scope: Frontend
+
+    In order to use the front-end libraries with upgraded dependencies, make sure to first upgrade your implementation
+    to Angular 14 by following the steps mentioned on this page.
+
+    *Redundant stylesheets*
```
Comment (marijnritense) [inline release-notes/major10/10.0.0/migration.md:34]: Maybe this makes more sense as an indented bullet? Same goes for Matching dependencies.

File: release-notes/major10/10.0.0/migration.md [hunk 2]
```diff
@@ -24,4 +24,99 @@ This page describes how to update Valtimo from the previous version to the curre
...
+             useFactory: MultiTranslateHttpLoaderFactory,
+             deps: [HttpBackend, ConfigService],
+          },
+        }),
+      ],
+      ...
+    })
+    ...
+    ```
+    Next, if your implementation uses custom translation resources, these need to be adjusted in the environment file
```
Comment (marijnritense) [inline release-notes/major10/10.0.0/migration.md:111]: `Next, ` can be removed

File: release-notes/major10/10.0.0/migration.md [hunk 3]
```diff
@@ -24,4 +24,99 @@ This page describes how to update Valtimo from the previous version to the curre
...
+    ```
+  
+    Note: if your implementation has any dependencies not included in the above example, and installing dependencies
+    or building your application causes errors, make sure the versions of these dependencies match the versions
+    in this release of `valtimo-frontend-libraries`.
+
+    *Translate loader upgrade*
+
+    `ngx-translate-multi-http-loader` has been upgraded from version 7 to version 9. First of all, the import of
+    `TranslateModule` in the file `app.module.ts` in your implementation needs to be adjusted. More specifically,
```
Comment (marijnritense) [inline release-notes/major10/10.0.0/migration.md:86]: These two sentences feel like you're speaking to someone more than writing down instructions. Could you rephrase it so it's a bit more to the point?

**Review by Klaas-Ritense — APPROVED** [review]

**Review by marijnritense — APPROVED** [review]
